backend/graph/schema.graphqls

```diff
@@ -27,14 +31,13 @@ type Process {
 }
 
 type Instance {
-  activeInstances: Int!
   bpmnLiveStatus: String!
   bpmnResource: String!
-  completedInstances: Int!
-  deploymentTime: DateTime!
+  startTime: DateTime!
   processId: Int!
-  processKey: Int!
+  instanceKey: Int!
   version: Int!
+  status: String!
```

Comment:
Is this `status` the same as `bpmnLiveStatus` or different?

Reply:
different. `status` should be "Active" or "Completed"